tgstation.dme

```diff
@@ -37,6 +37,7 @@
 #include "code\__DEFINES\clockcult.dm"
 #include "code\__DEFINES\colors.dm"
 #include "code\__DEFINES\combat.dm"
+#include "code\__DEFINES\components.dm"
```

Comment:
You put it in the legacy file rather than the current codebase file. Including this file will probably result in a lot of bad stuff. The proper files are `code/__DEFINES/dcs/flags.dm` and `code/__DEFINES/dcs/signals.dm`.